draft-holmberg-mmusic-t140-usage-data-channel.xml

```diff
@@ -185,6 +184,41 @@
 
             ]]></artwork>
           </figure>
+          <t>
+            Below is an example of an m= section of an offer 
+            for a T.140 data channel offering real-time text
+            conversation in Spanish and Esperanto, and an m= section
+            in the associated answer accepting Esperanto.
+          </t>
+          <figure>
+            <preamble></preamble>
+            <artwork><![CDATA[
+
+Offer:
+
+    m=application 911 UDP/DTLS/SCTP webrtc-datachannel
+    c=IN IP6 2001:db8::3
+    a=max-message-size:1000
+    a=sctp-port 5000
+    a=dcmap:1 label="ACME customer service";subprotocol="t140"
+    a=dcsa:1 fmtp:-
+    a=dsca:1 hlang-send:es eo
+    a=dsca:1 hlang-recv:es eo
+
+Answer:
+
+    m=application 911 UDP/DTLS/SCTP webrtc-datachannel
+    c=IN IP6 2001:db8::3
```

Comment:
I think the answer should have another c= address than the offer. please invent another address.

Reply:
I was going to do that, but forgot :) Will fix.